docs/multiple_inputs.md

```diff
@@ -0,0 +1,251 @@
+This tutorial works through such a scenario.
+
+
+We have partitioned the data contained within the main example dataset into two sets:
+1. An "Australian" dataset, containing 91 genomes from Victoria, Australia. These were genomes uploaded to NCBI from [Torsten Seemann et al.,](https://www.doherty.edu.au/people/associate-professor-torsten-seemann) but are used in this tutorial to represent a small subset of genomes which may not yet be public.
+2. A "worldwide" dataset for context. Often this would be the entire NCBI/GISAID dataset, but here only includes 327 genomes for speed and data-sharing reasons. 
+
+
+Our aim is to produce an analysis of the 91 Australian genomes with select worldwide genomes for context. To achieve this, we wish to apply different input-dependent filtering, subsampling and colouring steps.
+
+
+
+## Overview of the files used in this tutorial
+
+The **sequences and metadata** for this tutorial are in `data/example_multiple_inputs.tar.xz` and must be decompressed via `tar xf data/example_multiple_inputs.tar.xz --directory data/`.
+
+You should now see the following starting files:
+```sh
+data/example_metadata_aus.tsv           # Aus data (n=91) from Seemann et al.
+data/example_sequences_aus.fasta
+data/example_metadata_worldwide.tsv     # Worldwide, contextual data (n=327)
+data/example_sequences_worldwide.fasta
+```
+
+The files are small enough to be examined in a text editor -- the format of the worldwide metadata is similar to the `nextmeta.tsv` file which you may download from GISAID, whereas the format of the Australian metadata is more limited, only containing sampling date and geographic details, which may be more realistic for a newly generated sequencing run.
+Note: see `data/example_metadata.tsv` for the full metadata of these Australian samples, we've intentionally restricted this here to mimic a real-world scenario.
+
+
+The **build-specific configs** etc are in `my_profiles/example_multiple_inputs`
+
+```sh
+my_profiles/example_multiple_inputs/config.yaml
+my_profiles/example_multiple_inputs/builds.yaml # this is where the input files and parameters are specified
+my_profiles/example_multiple_inputs/my_auspice_config.json
+```
+
+
+## Setting up the config
+
+Typically, inside the `builds.yaml` one would specify input files such as
+
+```yaml
+# traditional syntax for specifying starting files
+sequences: "data/sequences.fasta"
+metadata: "data/metadata.tsv"
+```
+
+For multiple inputs, we shall use the new `inputs` section of the config to specify that we have two different inputs, and we will give them the names "aus" and "worldwide":
+
+```yaml
+# my_profiles/example_multiple_inputs/builds.yaml
+inputs:
+  aus: 
+    metadata: "data/example_metadata_aus.tsv"
+    sequences: "data/example_sequences_aus.fasta"
+  worldwide:
+    metadata: "data/example_metadata_worldwide.tsv"
+    sequences: "data/example_sequences_worldwide.fasta"
```

Comment:
Shifted to this approach in 99bd0f5, and updated the schema to enforce the "name" and "metadata" properties. (As an aside, we could be using the schema a lot more 😉). Internally, we convert this to an `OrderedDict` straightaway so that we can still access inputs by "name" keyword throughout the rules.

Reply:
Awesome! Thanks, @jameshadfield!